src/inverseRobot.js

```diff
@@ -7,7 +7,16 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const result = {};
+
+  for (const key in robot) {
+    if (result.hasOwnProperty(robot[key])) {
+      return null;
+    }
+    result[robot[key]] = key;
```

Comment:
`robot[key` is being used in two places, it should be a new constant ;)
Furthermore - it will look way cleaner.